.github/workflows/infrastructure.yml

```diff
@@ -212,7 +267,49 @@ jobs:
             --sku-name ${{ vars.APIM_PLAN }} \
             --tags ${{ env.tags }}
 
-  # 3. Container app configuration
+  # 3. Network configuration
+  network:
+    name: Network 🛂
+    needs: base
+    environment: ${{ needs.base.outputs.environment }}
+    runs-on: [self-hosted, APIM, infrastructure]
+    steps:
+      - name: Pre-production 💫
+        if: contains('["dev", "staging"]', env.target)
+        run: echo "type=Preproduction" >> $GITHUB_ENV
+
+      - name: Production 💫
+        if: ${{ 'prod' == env.target }}
+        run: echo "type=Production" >> $GITHUB_ENV
+
+      - name: Tags 🏷️
+        run: echo tags='Environment=${{ env.type }}' \
+          'Product=${{ env.product }}' \
+          'Team=development' >> $GITHUB_ENV
+
+      - name: Login 🔐
+        uses: azure/login@v1
+        with:
+          creds: ${{ secrets.AZURE_CREDENTIALS }}
+
+      - name: Azure defaults ✨
+        uses: Azure/cli@v1.0.7
+        with:
+          inlineScript: |
+            # Basic
+            az configure --defaults location=${{ vars.REGION }}
+            az configure --defaults group=rg-${{ env.product }}-${{ env.target }}-${{ vars.VERSION }}
+
+      - name: Subnet routing table 🌐
+        uses: Azure/cli@v1.0.7
+        with:
+          inlineScript: |
+            # Associate UKS route table to default subnet
+            az network vnet subnet update \
+            --name snet-${{ env.product }}-${{ vars.VERSION }} \
+            --route-table route-${{ env.product }}-uks
```

Comment:
should `uks` be here? or we need to add it to secrets?

Reply:
Its fine to add `UKS` here it does not imply anything private.